src/backend/marsha/core/api/live_registration.py

```diff
@@ -118,6 +128,31 @@ def get_throttles(self):
 
         return [throttle() for throttle in throttle_class]
 
+    def perform_create(self, serializer):
+        """Overrides perform_create to send the mail and catch error if needed"""
+        super().perform_create(serializer)
+        try:
+            msg_plain = render_to_string("core/mail/text/sample.txt")
+            msg_html = render_to_string("core/mail/html/sample.html")
+            send_mail(
+                _("mail_registration_object"),
+                msg_plain,
+                settings.EMAIL_FROM,
+                [serializer.validated_data["email"]],
+                html_message=msg_html,
+                fail_silently=False,
+            )
+        except smtplib.SMTPException as exception:
+            # no exception raised as user can't sometimes change his mail,
+            # we set should_reminders to false not to send reminders in error
+            logger.warning(
+                "registration mail %s not send %s",
+                serializer.validated_data["email"],
+                str(exception),
+            )
+            serializer.validated_data["should_send_reminders"] = False
```

Comment:
I would not change this to `False`. It can be a temporary failure. Otherwise there is no interest to register this user. With a retry strategy (not in the scope of this PR) this case can be managed.

Reply:
So I put it back to fail_silently True, otherwise, I don't see the point at this time of catching the error wdyt ?

Reply:
Logging the error ?

Reply:
yes meanwhile